tests/console/consoletest_setup.pm

```diff
@@ -70,7 +70,10 @@ sub run {
     disable_and_stop_service('packagekit.service', mask_service => 1);
 
     # init
-    check_console_font if has_ttys();
+    if (has_ttys()) {
+        check_console_font;
+        script_run '. /etc/bash.bashrc.local';
```

Comment:
Needs a comment that this applies to the console switched to by `check_console_font` now